### PR TITLE
docs: updates for 0.13.2

### PR DESCRIPTION
**Pull request by jessicaschilling**
Changelog and readme updates for v0.13.1, including addition of manual testing steps.

@lidel - can you please fill in the details for protocol handler test? Afraid my memory is sketchy ...

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@lidel - added manual testing notes here since we're already editing the readme. Can you please fill in the details for protocol handler test? Afraid my memory is sketchy ...

File: README.md [hunk 1]
```diff
@@ -118,6 +118,11 @@ Other languages are periodically pulled from [Transifex](https://www.transifex.c
 
 ## Releasing
 
+- Manually test a few things that don't transfer well to automated testing:
```
Comment (rafaelramalho19) [inline README.md:121]: I would move this to a section called Pre-release testing? Otherwise it gets too lengthy

Comment (jessicaschilling) [reply README.md:121]: @rafaelramalho19 - yep, planning to rewrite all this stuff under https://github.com/ipfs-shipyard/ipfs-desktop/issues/1670 so will leave as is for now if you don't mind. 😊

Comment (rafaelramalho19) [reply README.md:121]: Alright 👍

**Review by rafaelramalho19 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -118,6 +118,11 @@ Other languages are periodically pulled from [Transifex](https://www.transifex.c
...
+     - Mac/Win/Linux: confirm that "Take Screenshot" under menubar/system tray menu works as expected for both single- and multi-monitor setups (file(s) imported, correct link copied to clipboard)
+     - Mac only: drag/drop onto menubar icon behaves as expected when dragging one file, several files, and a combination of files/folders (file(s) imported, correct link copied to clipboard)
+     - Win only: right-click on a file and "Add to IPFS" from context menu works as expected (file imported, correct link copied to clipboard)
+     - Mac/Win: confirm that OS-wide protocol handler got registered by opening <a href="ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi">`ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi`</a> in user agent without IPFS Companion
```
Comment (lidel) [inline README.md:125]: @jessicaschilling seems that Github ignores `<a>` tags that use different protocols than http, so it renders as text – feel free to reword :pray:

File: README.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ Then follow the steps below to clone the source code, install the dependencies a
 git clone https://github.com/ipfs-shipyard/ipfs-desktop.git
 cd ipfs-desktop
 npm ci
-npm build
+npm run-script build
```
Comment (lidel) [inline README.md:108]:
```suggestion
npm run build
```

**Review by lidel — APPROVED** [review]
lgtm, thank you!

**Pull request merged**